src/CompositeFixtureLoader.php

```diff
@@ -0,0 +1,32 @@
+<?php
+namespace Lakion\ApiTestCase;
+
+use Nelmio\Alice\Fixtures;
+
+class CompositeFixtureLoader
```

Comment:
final. Also, this class seems to be something totally not related to this project